<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtQQ_ServerClass</class>
 <widget class="QDialog" name="QtQQ_ServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>502</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>999</width>
    <height>502</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>999</width>
    <height>502</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtQQ_Server</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>698</width>
      <height>498</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>700</width>
       <height>500</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>500</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>702</x>
     <y>1</y>
     <width>296</width>
     <height>500</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>0</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>tiger.png</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>201</y>
      <width>291</width>
      <height>290</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>291</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>员工部门：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>员工姓名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>员工寸照：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="employeeDepBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>6</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>人事部</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>研发部</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>市场部</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>36</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入员工姓名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectPictureBtn">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>66</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>新增员工</string>
     </property>
    </widget>
    <widget class="QLabel" name="headLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>  员工寸照</string>
     </property>
    </widget>
    <widget class="QComboBox" name="departentBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>191</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>公司群</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>人事部</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>研发部</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>市场部</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="queryDepartmentBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>189</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>查询员工</string>
     </property>
    </widget>
    <widget class="QPushButton" name="queryIdBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>227</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>查询员工</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="queryIdLineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>227</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入员工id号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>261</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>注销员工</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="logoutIdLineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入员工id号</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtQQ_Server.qrc"/>
 </resources>
 <connections/>
</ui>
